src/patch.css

```diff
@@ -0,0 +1,94 @@
+/*-----------------------------------------------------------------------------
```

Comment:
I believe we need to include JupyterLab's license in this repo if we copy code from there.

Reply:
yes, you're right. I really wonder if we need this patch though? I think I'll do some more testing with it removed, and if it stays we'll bring the in license.